dev/ory/jwks.json

```diff
@@ -0,0 +1,18 @@
+{
+  "keys": [
```

Comment:
note @bodymindarts a private key should be dynamically generated here. 

this can be done with `  oathkeeper credentials generate --alg RS256`

Reply:
Will have to research in the chart how to pre-provision it and generate in terraform

Reply:
DO we want the id_token https://github.com/ory/k8s/blob/e3c6951bdb22f7216b9f55fea751ca58df86a6c3/docs/helm/oathkeeper.md#json-web-key-set-for-mutator-id_token or the authenticator jwks https://github.com/ory/k8s/blob/e3c6951bdb22f7216b9f55fea751ca58df86a6c3/docs/helm/oathkeeper.md#json-web-key-set-for-authenticator?
I think the id_token right.

Reply:
id_token for the mutator

Reply:
https://github.com/GaloyMoney/galoy/pull/1653/files#diff-343ce5007f35549b95e3d0db237fce76fbce3130d06916a33423288f98fdd3b1R27